build.gradle

```diff
@@ -47,4 +48,19 @@ jar {
     manifest {
         attributes 'Main-Class': mainClassName
     }
-}
\ No newline at end of file
+}
+
+test {
+    ignoreFailures = true
+    jacoco {
+        append = false
+        destinationFile = file("$buildDir/jacoco/jacocoTest.exec")
+    }
+}
+
+jacocoTestReport {
+    reports {
+        xml.enabled true
```

Comment:
I have switched the test coverage from CSV to XML as it is much easier to work with:
- can collect both lines covered and total lines in one single command
- can reliably collect coverage at package level

This is the command I used to collect coverage:
```
xmllint --xpath '//package[@name="befaster/solutions/SUM"]/counter[@type="INSTRUCTION"]' build/reports/jacoco/test/jacocoTestReport.xml
```